src/constants.py

```diff
@@ -15,6 +15,9 @@
 LOG_DIR = PurePath("/var/log")
 LOG_FILE = LOG_DIR / "admin_ui.log"
 RULES_CONFIGMAP_FILE_NAME = "admin_ui_rules.json"
+OAUTH_SCOPES = "openid profile offline_access"
```

Comment:
small issue: unfortunately the library we use to parse env cnofig is kind of limited and only supports string slices coming from a string that is comma separated, so this needs to be `"openid,profile,offline_access"`

Reply:
Interesting, I will change it but it is weird that it worked like this without throwing an error

Reply:
I tried it before commenting, it interprets the env var as an array with one string with the whole thing. I haven't tried hydra behavior with a string like that, but go-oidc takes an array for the scopes so I imagine it would error out